hack/fetch-releases.sh

```diff
@@ -122,6 +122,27 @@ release_yaml_pac() {
 
     echo "Info: Added $comp/$fileName:$version release yaml !!"
     echo ""
+
+    runtime=( go java nodejs python )
+    for run in "${runtime[@]}"
+    do
+    	echo "fetching PipelineRun template for runtime: $run"
+
+    	source="https://raw.githubusercontent.com/sm43/pipelines-as-code/pac-pr-templates/pkg/cmd/tknpac/generate/templates/${run}-template.yaml"
```

Comment:
This should probably point to `openshift-pipelines/pipelines-as-code` ?

Reply:
yeah the PAC pr was not merged 😅 then!